### ISSUE TITLE
Non-commercial and no-derivatives included in the definition?

### ISSUE THREAD

**Issue by Ainali** [FOSSRIT/open-work-definition#6]
As the definition is written now, it looks like the non-commercial and no-derivatives variants of the Creative Commons licenses are included. But they are not generally seen as being open, as they heavily restrict what you may do with the works under such licenses. Could this be clarified?

**Comment by sjfossbox** [comment]
It's a good question. We'll need to think it through.  My first thought is that this touches on the academic and humanitarian spaces in addition to the industry. Some grant-funded work might require non-commercial, so I could see support for including those works.  My own preference would be to exclude non-derivative.

**Comment by Ainali** [comment]
Yes, please do think about it. I fear that if you're going with a definition that is not aligned with the established concepts of open, this well needed initiative might risk not gaining the favor of the existing open movement.

**Comment by sjfossbox** [comment]
Which Open movement?  Free Software? Free Culture? Open Core?  :-)

On Mar 16, 2023, at 1:31 PM, Jan Ainali ***@***.***> wrote:



Yes, please do think about it. I fear that if you're going with a definition that is not aligned with the established concepts of open, this well needed initiative might risk not gaining the favor of the existing open movement.

**Comment by Ainali** [comment]
We have the [Open Definition](https://opendefinition.org/) that is already being used across all these fields. Even Creative Commons themselves [puts the NC and ND in the "yellow"](https://creativecommons.org/share-your-work/public-domain/freeworks/). But perhaps we could reverse this? Does any of the listed organizations think restricting the use should be considered to be "open"?

**Comment by Nolski** [comment]
Thanks for the comment @Ainali . You definitely bring up a great (and certainly ongoing) concern about how dogmatic these definitions should be when utilizing the term "Open." We certainly found when working in many groups across academia, industry, non-profits, and governments that many key stakeholders had much broader use of the term and certainly had desires to consider works open even when they were targeting certain communities (such as research communities or humanitarian communities). 

A big reason we decided to document and define the term Open Work is that we found existing definitions to alienate many communities that could very much make use of methodologies employed by Open Work communities (such as InnerSource). A primary goal of this definition is to bring together communities that are not learning from each other when they could be.
